packages/react-core/src/components/AlertGroup/AlertGroupInline.tsx

```diff
@@ -7,9 +7,14 @@ export const AlertGroupInline: React.FunctionComponent<AlertGroupProps> = ({
   className,
   children,
   isToast,
+  isDynamic,
   ...rest
 }: AlertGroupProps) => (
-  <ul className={css(styles.alertGroup, className, isToast ? styles.modifiers.toast : '')} {...rest}>
+  <ul
+    aria-live={isDynamic ? 'polite' : null}
```

Comment:
In [Alert](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/Alert/Alert.tsx#L173-L176), we use the "belt and suspenders" approach, in case certain user agent doesn't default to the same implicit aria roles. Can we explicitly set aria-atomic here so it's consistent with the Alert strategy? Users can still override that default by passing aria-atomic in addition to isDynamic/isLiveRegion. The benefit is that they are getting sensible defaults without having to do a lot of research/testing on the subject.